orttraining/orttraining/python/ort_trainer.py

```diff
@@ -311,7 +311,7 @@ def convert_model_loss_fn_to_onnx(model, loss_fn, model_desc, device, inputs):
     torch.onnx._export(model, tuple(sample_inputs), f,
                        input_names=input_names,
                        output_names=output_names,
-                       opset_version=10,
+                       opset_version=11,
```

Comment:
I have been running with opset11 for some time, runs fine. Let me know if there's any issue upgrading this.

Reply:
This will also affect other models. Can you please start a email thread check compatibility with the team?

Also, converter team is working on opset 12 support already. What's the gap for support opset 12 with ORT training? 


---
In reply to: [402687351](https://github.com/microsoft/onnxruntime/pull/3411#discussion_r402687351) [](ancestors = 402687351)

Reply:
will do

Reply:
removed opset change in favor of [https://github.com/microsoft/onnxruntime/pull/3231](url)